src/command/call/cmd.ts

```diff
@@ -0,0 +1,9 @@
+import { Command } from "cliffy/command/mod.ts";
+import { engineCommand } from "../../execute/engine.ts";
+
+export const callCommand = new Command()
+  .name("call")
+  .action(() => {
+    callCommand.showHelp();
+    Deno.exit(1);
+  }).command("engine", engineCommand);
```

Comment:
If I understand this correctly, we will need a `description(...)` here. Otherwise `quarto call --help` doesn't do anything, right?

It'll necessarily be something generic like "access functions of Quarto subsystems such as 'engine'", and we'll update it as appropriate.